pytest_django/runner.py

```diff
@@ -0,0 +1,40 @@
+from argparse import ArgumentParser
+from typing import Any, Iterable
+
+
+class PytestTestRunner:
+    """Runs pytest to discover and run tests."""
+
+    def __init__(
+        self, verbosity: int = 1, failfast: bool = False, keepdb: bool = False, **kwargs: Any
+    ) -> None:
+        self.verbosity = verbosity
+        self.failfast = failfast
+        self.keepdb = keepdb
+
+    @classmethod
+    def add_arguments(cls, parser: ArgumentParser) -> None:
```

Comment:
```suggestion
    def add_arguments(cls, parser: ArgumentParser) -> None:
        parser.add_argument(
            "--failfast",
            action="store_true",
            help="Stops the test suite after the first failure.",
        )
```

Reply:
Keep in mind, i applied this, and had to remove it. `failfast` is a standard option already inside of the django `test`, duplicating it causes an error to be raised.

Reply:
The reason I suggested it is that I saw that Django's `DiscoverRunner` has it, so assumed we need it too. But if not, great!